discovery-client/src/main/java/io/micronaut/discovery/serde/InetAddressSerde.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright 2017-2023 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micronaut.discovery.serde;
+/*
+ * Copyright 2017-2023 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import io.micronaut.core.annotation.Internal;
+import io.micronaut.core.annotation.NonNull;
+import io.micronaut.core.annotation.Nullable;
+import io.micronaut.core.type.Argument;
+import io.micronaut.serde.Decoder;
+import io.micronaut.serde.Encoder;
+import io.micronaut.serde.Serde;
+import io.micronaut.serde.config.SerdeConfiguration;
+import jakarta.inject.Singleton;
+
+import java.io.IOException;
+import java.net.InetAddress;
+
+/**
+ * {@link Serde} implementation of {@link InetAddress}.
+ * This is a based on `com.fasterxml.jackson.databind.ser.std.InetAddressSerializer` which is licenced under the Apache 2.0 licence.
+ */
+@Internal
+@Singleton
+public class InetAddressSerde implements Serde<InetAddress> {
+    private final boolean asNumeric;
+
+    public InetAddressSerde(SerdeConfiguration serdeConfiguration) {
```

Comment:
```suggestion
    public InetAddressSerde() {
```

Reply:
I have removed the serde and use the one shipped in Micronaut Serde 2.5.6 https://github.com/micronaut-projects/micronaut-discovery-client/pull/523/commits/cadd5a08aa3cf03eee2a78397857db72d97fa709